src/models/legend.js

```diff
@@ -187,6 +182,44 @@ nv.models.legend = function() {
 
         return chart;
     }
+    
+    function toggleItem(d,i,data){
```

Comment:
the two function toggleItem and checkItemOnly are copied from old event for re-usage, between common click/dblclick and touchstart/taphold.